frontend/src/app/store/actions/addAction.js

```diff
@@ -0,0 +1,6 @@
+import { ADD_COURSE } from '../../constants/constants';
+
+export const addCourseAction = (course) => ({
+    type: ADD_COURSE,
+    payload: course
+});
```

Comment:
All the actions for a course should be put into a single file

Reply:
I'll keep that in mind